### PR TITLE
Ag manufacturing patches

### PR DESCRIPTION
**Pull request by sebassem**

### CODE REVIEW

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ This diagram shows the welding defect inference flow, which consists of five mai
     5. Map the index of the highest probability value to a corresponding class label.
     6. Draw the label containing the predicted class and its corresponding probability on the input image
 
-    Once the image is processed, is then served to the main application to render it to the user. Final image contains the result of the welding inference (**no weld**, **normal weld**, **porosity**) and the probability of the result. 
+    Once the image is processed, is then served to the main application to render it to the user. Final image contains the result of the welding inference (**no weld**, **normal weld**, **porosity**) and the probability of the result.
 
 ![Welding defect uI](./img/welding_ui.png)
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md:58]:
```suggestion
![Welding defect UI](./img/welding_ui.png)
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md [hunk 2]
```diff
@@ -77,6 +77,26 @@ Expected color order is BGR.
 
 The features is a blob with the shape 1, 3 containing probability scores for three output classes (**no weld**, **normal weld** and **porosity**).
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md:78]: something about this sentence seems off.

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md [hunk 3]
```diff
@@ -1,11 +1,55 @@
...
 type: docs
 weight: 5
-title: WIP
-linkTitle: WIP
+title: Enabling AI at the Edge to enhance workers safety
+linkTitle: Enabling AI at the Edge to enhance workers safety
 summary: |
-  WIP
-serviceOrPlatform: WIP
+    The enabling AI at the Edge to enhance workers safety page provides an overview of how Contoso Motors leverages AI to ensure workers' safety by detecting workers with no helmets on the factory floor. It describes the architecture and flow of information for detecting and classifying helmet adherence using AI. The page also explains the steps involved in the inference process, including UI selection, RTSP video simulation, frame capturing, image pre-processing/inferencing, and post-processing/rendering.
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md:7]:
```suggestion
    The enabling AI at the Edge to enhance workers safety page provides an overview of how Contoso Motors uses AI to ensure workers' safety by detecting workers with no helmets on the factory floor. It describes the architecture and flow of information for detecting and classifying helmet adherence using AI. The page also explains the steps involved in the inference process, including UI selection, RTSP video simulation, frame capturing, image pre-processing/inferencing, and post-processing/rendering.
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md [hunk 4]
```diff
@@ -1,11 +1,55 @@
...
+  - AI
+  - AKS EDGE ESSENTIALS
+  - RTSP
+---
+
+# Enabling AI at the Edge to enhance workers safety
+
+## Overview
+
+Contoso Motors uses AI-enhanced computer vision to improve workers' safety by detecting workers with no helmets on the factory floor. Workers safety is one of the four computer vision use cases that Contoso Motors uses, which also include object detection, and human pose estimation. While each use case has its own unique characteristics, they all follow the same inferencing architecture pattern and data flow.
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md:21]:
```suggestion
Contoso Motors uses AI-enhanced computer vision to improve workers' safety by detecting workers with no helmets on the factory floor. Worker safety is one of the four computer vision use cases that Contoso Motors uses, which also include object detection, defect detection, and human pose estimation. While each use case has its own unique characteristics, they all follow the same inferencing architecture pattern and data flow.
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md [hunk 5]
```diff
@@ -1,11 +1,55 @@
...
+
+## Architecture
+
+![Welding defect archietcture](./img/flow.png)
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md:25]:
```suggestion
![Welding defect architecture](./img/flow.png)
```

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md [hunk 6]
```diff
@@ -1,11 +1,55 @@
...
+
+### Model
+
+#### Inputs
+
+#### Outputs
+
+## Operation technology (OT) Manager Experience
+
+Contoso leverages their AI-enhanced computer vision to monitor the safety helmet adherence for workers on the factory floor to help OT managers ensure workers safety through the "Control Center" interface.
```
Comment (dkirby-ms) [inline docs/azure_jumpstart_ag/manufacturing/contoso_motors/workers_safety/_index.md:35]:
```suggestion
Contoso uses AI-enhanced computer vision to monitor the safety helmet adherence for workers on the factory floor to help OT managers ensure workers safety through the "Control Center" interface.
```

**Review by dkirby-ms — APPROVED** [review]

File: docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md (hunk 2 above)
Comment (sebassem) [reply docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md:78]: @fcabrera23 can you help with explaining what this sentence should say ?

**Pull request merged**